### DISCUSSION TITLE
Transparent overlay of egui app on the web

### DISCUSSION THREAD

**Discussion by moritztng** [emilk/egui#3283] [Q&A] [answered]
When I try to overlay the egui canvas on top of the html page the background is not completely transparent. How can I make the background of the egui app transparent?

Egui app not completely transparent:
![Screenshot 2023-08-28 12 55 35](https://github.com/emilk/egui/assets/19519902/ee81adc5-ffb3-4c07-b0a1-02fe72d6ae6c)
Color of html page:
![Screenshot 2023-08-28 12 55 59](https://github.com/emilk/egui/assets/19519902/c2d36361-a7eb-485d-92e4-5345ff26f447)

Thank you!

**Comment by emilk** [comment, marked as answer]
https://docs.rs/eframe/latest/eframe/trait.App.html#method.clear_color

**Reply by moritztng** [reply]
thanks! I receive a webrtc videostream and originally drew it on a canvas, got the data and painted it as a texture on the app. Unfortunately that is pretty inefficient. So i'll just use a html video element and overlay egui on top of it. would you say that this is the right approach?
